exporter/exporterhelper/internal/bounded_memory_queue_test.go

```diff
@@ -127,9 +129,15 @@ func TestShutdownWhileNotEmpty(t *testing.T) {
 	q.Produce(newStringRequest("i"))
 	q.Produce(newStringRequest("j"))
 
+	go func() {
+		require.EventuallyWithT(t, func(c *assert.CollectT) {
+			assert.False(c, q.Produce(newStringRequest("x")), "cannot push to closed queue")
```

Comment:
This `Produce` always returns false due to queue overflow. We probably need to significantly increase the capacity to ensure it's rejected due to the stopped queue

Reply:
The queue has one slot open because one item is considered read as it is blocked in processing. 9 items are in the channel. The workaround imo is to add a few more consumers, and we can even assert on the size of the queue based of those. Please see my changes.

Reply:
Since we send a request every 10 milliseconds for 1 second (100 possible requests in total), even with 5 workers, we potentially can run into the queue overflow, especially on slow windows. I'd suggest setting 1000 queue capacity to make sure it's always rejected due to stopped queue

Reply:
that makes sense, done.